tools/autograd/gen_autograd_functions.py

```diff
@@ -53,6 +53,10 @@
     ${release_variables}
   }
   ${will_release_variables}
+#ifdef COMPILED_AUTOGRAD
```

Comment:
Annoying that we have this windows limitation :(

Reply:
Fixed in #105286